sdk/python/README.rst

```diff
@@ -46,7 +46,7 @@ To use from Go, use ``go get`` to grab the latest version of the library
 
 ::
 
-    $ go get github.com/pulumi/pulumi-f5bigip/sdk/go/...
+   $ go get github.com/pulumi/pulumi-f5bigip/sdk/go/...
```

Comment:
I've noticed these going backward and forward for me - and frequently breaking CI after I re-gen on my laptop.  @ellismg Any idea what this formatting difference in `.rst`/pandoc is about?  I've tried to make sure I'm using the same version - but I still consistently see issues with this file.